pkg/storage-init/storage-init.sh

```diff
@@ -233,6 +240,8 @@ if P3=$(findfs PARTLABEL=P3) && [ -n "$P3" ]; then
             echo "$(date -Ins -u) Cannot import persist pool on P3 partition $P3 of type $P3_FS_TYPE, recreating it as $P3_FS_TYPE_DEFAULT"
             INIT_FS=1
             P3_FS_TYPE="$P3_FS_TYPE_DEFAULT"
+        else
+            zfs_adjust_features
```

Comment:
Here you set it on a normal boot. Does that cause any failures or error logs if the device falls back to an earlier EVE-OS version? You said you tested downgrade but were there error logs after the downgrade?

Reply:
With new version we will see:
```
08189017-8dd1-45fa-9676-733c38e2f6a1:~# zpool get feature@draid persist
NAME     PROPERTY       VALUE          SOURCE
persist  feature@draid  enabled        local
```
and not complain in `zpool status`

When we downgrade from the new version installation (or using features adjustment across old pool), we will see:
```
08189017-8dd1-45fa-9676-733c38e2f6a1:~# zpool get all|grep draid
persist  unsupported@org.openzfs:draid  inactive                       local
```
and not complain in `zpool status`

If we will not adjust features, then after upgrade to the new version from the old installation (for example 8.7.0) we will see:
```
c97475ea-67e5-42f9-bc32-c231caa16199:~# zpool get feature@draid persist
NAME     PROPERTY       VALUE          SOURCE
persist  feature@draid  disabled       local

c97475ea-67e5-42f9-bc32-c231caa16199:~# zpool status
  pool: persist
 state: ONLINE
status: Some supported and requested features are not enabled on the pool.
        The pool can still be used, but some features are unavailable.
action: Enable all features using 'zpool upgrade'. Once this is done,
        the pool may no longer be accessible by software that does not support
        the features. See zpool-features(7) for details.
config:

        NAME        STATE     READ WRITE CKSUM
        persist     ONLINE       0     0     0
          sda9      ONLINE       0     0     0

errors: No known data errors
```

Let me update PR description